src/components/Navigation/index.js

```diff
@@ -0,0 +1,139 @@
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+
+import styled from 'styled-components';
+import { Icon } from 'semantic-ui-react';
+
+const StyledMobileBar = styled.div`
```

Comment:
I would extract style-related code to something like `Navigation/styles.js` and would leave navigation-related code here